src/accounts/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
+
+import Wallet from "./wallet";
+import axios from 'axios';
+import AddWalletModal from "../modals/addWallet";
+import Loader from "../components/shared/Loader";
+import Swal from "sweetalert2";
+
+import "./style.scss"
+
+export default function Accounts() {
+
+    const [accountDB, setAccountDB] = useState([]);
+    const [modalIsOpen, setModalIsOpen] = useState(false)
+    const [updateDB, setSpdateDB] = useState(false)
+    const [isLoaded, setIsLoaded] = useState(false)
+    const [isError, setIsError] = useState(false)
+
+    function ErrorAlert() {
+        Swal.fire({
+            icon: 'error',
+            title: 'Oops...',
+            text: 'Something went wrong!',
+            footer: '<a href="/">try Again ?</a>'
+        })
+    }
+    useEffect(() => {
+        const getData = async () => {
```

Comment:
remove this function as you are doing `.then().catch()`

Reply:
done